BUILD.bazel

```diff
@@ -19,6 +19,7 @@ java_plugin(
 # google-java-format
 java_binary(
     name = "google_java_format_binary",
+    # Not letting he formatter to consume too much memory (essential for CI builds)
```

Comment:
Nit: s/Not letting he formatter to consume too much memory/Not letting the formatter consume too much memory/

Reply:
Sorry, corrected.